dirty_cat/datetime_encoder.py

```diff
@@ -43,6 +46,12 @@ def __init__(self,
         self.extract_until = extract_until
         self.add_day_of_the_week = add_day_of_the_week
 
+    def _more_tags(self):
+        """
+        Used to ease the sklearn checks of the estimator.
```

Comment:
Is it used internally by sklearn ?
If so, I think it would be a good thing to mention it, something like

```suggestion
        Used internally by sklearn to ease the estimator checks.
```

(also applicable to the 3 other instances of this method in this PR)